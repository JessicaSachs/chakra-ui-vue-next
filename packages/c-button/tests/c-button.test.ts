import { render } from '@chakra-ui/vue-test-utils'
import CButton from '../src'

it('should render properly', () => {
  const { html } = render(CButton)
  expect(html()).toMatchSnapshot()
})
