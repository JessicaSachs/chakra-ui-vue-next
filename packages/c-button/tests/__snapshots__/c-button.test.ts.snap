// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render properly 1`] = `
<DocumentFragment>
  <button
    class="chakra-button css-0"
  />
</DocumentFragment>
`;
